int32 id     #description id
string type  # model type (name)
---
